### PR TITLE
100: refactor for relateddisplays

### LINKED ISSUE #100: Update RelatedDisplays component

### PR DESCRIPTION
**Pull request by Incin**

### CODE REVIEW

File: src/components/RelatedDisplays/RelatedDisplays.tsx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import { Position, Rnd } from 'react-rnd';
+import DisplayList from '../DisplayList';
+import { relatedDisplayGroupsSelector, selectedRelDispsSelector } from '../../selectors/display';
+import { contentHeightSelector, contentWidthSelector } from '../../selectors/ui';
+import { selectedDisplaySelector, displayListSelector } from '../../selectors';
+import { setRelDispPositions } from '../../actions';
+import uiConsts from '../../assets/styles/uiConsts';
+
```
Comment (dpenciso) [inline src/components/RelatedDisplays/RelatedDisplays.tsx:26]: did you mean to have this space?

File: src/components/RelatedDisplays/RelatedDisplays.tsx [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+const fixCSS = (value: string) => parseInt(value.replace('px', ''), 10);
+
+const boxStyle = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  border: 'solid 1px #ddd',
+  background: 'rgba(69, 138, 249, 0.4)',
+};
```
Comment (dpenciso) [inline src/components/RelatedDisplays/RelatedDisplays.tsx]: can this be moved into the scss file?

File: src/components/RelatedDisplays/RelatedDisplays.tsx [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+  };
+
+  const parentBoundary = {
+    position: 'relative',
+    background: '#efefef',
+    width: previewHeight * (contentWidth / contentHeight),
+    height: previewHeight,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  } as CSSProperties;
```
Comment (dpenciso) [inline src/components/RelatedDisplays/RelatedDisplays.tsx:98]: can this be moved into the scss file?

File: src/components/RelatedDisplays/RelatedDisplays.tsx [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+            className={styles.relatedDisplaysBadge}
+            classes={{ badge: styles.relatedDisplaysBadgeCircle }}
+            badgeContent={selectedRelDisps.length}
+          />
+        </i>
+      </button>
+      <Dialog
+        open={open}
+        className="trelliscope-app"
+        style={{ zIndex: 8000, fontWeight: 300 }}
```
Comment (dpenciso) [inline src/components/RelatedDisplays/RelatedDisplays.tsx]: can this be moved into the scss file?

File: src/components/RelatedDisplays/RelatedDisplays.tsx [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+                <StepButton onClick={() => setStep(1)} disabled={activeStep === 0 && selectedRelDisps.length === 0}>
+                  Arrange Panel Layout
+                </StepButton>
+              </Step>
+            </Stepper>
+            <div>{stepContent[activeStep]}</div>
+          </div>
+        </DialogContent>
+        <DialogActions>
+          <div style={{ textAlign: 'center', width: '100%' }}>
```
Comment (dpenciso) [inline src/components/RelatedDisplays/RelatedDisplays.tsx]: can this be moved into the scss file?

File: src/components/RelatedDisplays/RelatedDisplays.tsx [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+            <Button
+              // variant="outlined"
+              style={{ width: 200 }}
```
Comment (dpenciso) [inline src/components/RelatedDisplays/RelatedDisplays.tsx]: can this be moved into the scss file?

File: src/components/RelatedDisplays/RelatedDisplays.tsx [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+          {d.name}
+        </Rnd>
+      ))}
+    </div>,
+  ];
+
+  return (
+    <div>
+      <button type="button" onClick={handleOpen} className={styles.relatedDisplaysButton} style={propStyles.button}>
+        <i className="icon-open-add" style={{ paddingLeft: 2, lineHeight: '45px' }}>
```
Comment (dpenciso) [inline src/components/RelatedDisplays/RelatedDisplays.tsx]: can this be moved into the scss file?

**Review by dpenciso — CHANGES_REQUESTED** [review]
made some comments about putting styles in the scss file

File: src/components/RelatedDisplays/RelatedDisplays.tsx (hunk 1 above)
Comment (Incin) [reply src/components/RelatedDisplays/RelatedDisplays.tsx:26]: yes

File: src/components/RelatedDisplays/RelatedDisplays.tsx (hunk 3 above)
Comment (Incin) [reply src/components/RelatedDisplays/RelatedDisplays.tsx:98]: no it uses props for contentwidth and content height that are passed in the from the parent component.

Comment (dpenciso) [reply src/components/RelatedDisplays/RelatedDisplays.tsx:98]: ah yes you're right

File: src/components/RelatedDisplays/RelatedDisplays.tsx (hunk 1 above)
Comment (dpenciso) [reply src/components/RelatedDisplays/RelatedDisplays.tsx:26]: ok cool

**Review by dpenciso — APPROVED** [review]
:)

**Pull request merged**